frontend/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "yarn workspace @streamlit/app start",
     "build": "yarn workspace @streamlit/lib build && yarn workspace @streamlit/app build",
     "buildFast": "yarn buildLib && yarn workspace @streamlit/app buildFast",
-    "buildLib": "yarn workspace @streamlit/lib build",
+    "buildLib": "echo If you want to hot reload lib code, from the frontend dir, run: \n echo \"yarn watchLib\" && yarn workspace @streamlit/lib build",
```

Comment:
I wonder if this echo should also go into the `lib/package.json`

Reply:
These instructions are no longer correct, right?